sources/web/datalab/fileSearch.ts

```diff
@@ -131,7 +132,8 @@ function filter(pattern: string, data: string[]): string[] {
  */
 export function createHandler(settings: common.AppSettings): http.RequestHandler {
   appSettings = settings;
-  if (appSettings.enableFilesystemIndex) {
+  // Always enable the filesystem index on GCP VMs
```

Comment:
I think we still want to be able to disable this on a VM.

I'd rather that the default be unspecified and we pick a different value on a VM vs. off of one. Then, if the setting has been explicitly set then we honor that setting.

Reply:
Done.